src/main/java/com/prgms/vouchermanager/repository/voucher/MemoryVoucherRepository.java

```diff
@@ -0,0 +1,29 @@
+package com.prgms.vouchermanager.repository.voucher;
+
+import com.prgms.vouchermanager.domain.voucher.Voucher;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Repository
+@Profile("memory")
+public class MemoryVoucherRepository implements VoucherRepository{
+
+    private Map<UUID,Voucher>vouchers;
+
```

Comment:
```suggestion
    private final Map<UUID, Voucher> vouchers;
```

필드는 private final 로 시작하시길 권장합니다.